webui/react/src/services/apiConfig.ts

```diff
@@ -120,6 +120,23 @@ export const getUsers: DetApi<
   ),
 };
 
+export const postUser: DetApi<
+  Service.PostUserParams, Api.V1PostUserResponse, Api.V1PostUserResponse
+> = {
+  name: 'postUser',
+  postProcess: (response) => response,
+  request: (params) => detApi.Users.postUser(
+    {
+      user: {
+        active: true,
```

Comment:
**silly question:** probably good to be explicit, but wondering if `active: true` is needed and if not does the API call fail?

Reply:
Very good question, our current design of backend defaults `active` to `false` if not presented, I assume we'd like new users to be active.

Reply:
ah interesting, good find!